apax/config/optuna_config.py

```
@@ -41,7 +60,7 @@ def get_sampler(name: str) -> Type[optuna.samplers.BaseSampler]:
             return optunahub.load_module("samplers/auto_sampler").AutoSampler
         except ImportError as e:
             raise ImportError(
-                f"pruner {name} requires optunahub. Set pruner to other or install optunahub"
+                f"sampler {name} requires optunahub. Set pruner to other or install optunahub"
             ) from e
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)